pamelalozano/src/main/java/com/google/sps/servlets/Auth.java

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
nit: space?

Reply:
Emm does this means I should delete it? Or add more space? haha